server/models/imageUpload.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require("mongoose");
+
+const imageUploadSchema = new mongoose.Schema({
+  imageUrlPath: {
+    type: String,
+    default: "",
+    },
+});
+
+module.exports = ImageUrl = mongoose.model("imageUrl", imageUploadSchema);
```

Comment:
Let's rename the file and the schema to match the model name. Also, please use `PascalCase` for your model names:
```suggestion
module.exports = ImageUrl = mongoose.model("ImageUrl", imageUploadSchema);
```